enhancements/microshift/default-csi-plugin.md

```diff
@@ -0,0 +1,668 @@
+  by other parties than the authoring team (including upstream resources), and, if yes, how?
+  Please add those other parties as reviewers to the enhancement.
+
+  Examples:
+  - Adds a finalizer to namespaces. Namespace cannot be deleted without our controller running.
+  - Restricts the label format for objects to X.
+  - Defaults field Y on object kind Z.
+
+Fill in the operational impact of these API Extensions in the "Operational Aspects
+of API Extensions" section.
+--->
+
+**Start MicroShift with a user-provided ODF-LVM configuration**
+
+> After the device owner has logged into the device ... 
+
+1. The device owner will create a file called `/etc/microshift/lvmd.yaml`
+1. The device owner will define the lvmd config, according to its 
+[documentation](https://github.com/red-hat-storage/topolvm/blob/main/docs/lvmd.md)
+    
+    _For example:_
+
+        [root@ushift ~]# cat <<EOF > /etc/microshift/lvmd.yaml
+        socket-name: 
+        deviceClasses:
+        - default: true
+          name: "user-provided-config"
+          volume-group: rhel
+          spare-gb: 10 # in GB
+        socket-name: /run/lvmd/lvmd.sock # socket-name must always be /run/lvmd/lvmd.sock
+
+
+1. The device owner will start the microshift service, and proceed to validate the platform as described in **Validate 
+ODF-LVM function**
+1. The device owner will validate the system, following workflow **Validate ODF-LVM function**
+
+**Start MicroShift with a *malformed* user-provided ODF-LVM configuration**
+
+> After the device owner has logged into the device ...
+
+1. The device owner creates an incorrect lvmd config file under /etc/microshift/lvmd.config
+
+    _For example:_
+
+        [root@ushift ~]# cat <<EOF > /etc/microshift/lvmd.yaml
+        socket-name: 
+        deviceClasses:
+        - default: true
+          name: "user-provided-config"
+          volume-group: rhel
+          spare-gb: 10 # in GB
+        socket-name: /run/lvmd/lvmd.sock
+
+**Correct a _malformed_ user-provided ODF-LVM configuration and restart MicroShift**
+
+> After the device owner has logged into the device ...
+
+1. The device owner will edit the /etc/microshift/lvmd.yaml file to correct the error
+1. The device owner will restart MicroShift: `systemctl restart microshift`
+1. The device owner will validate the system, following workflow **Validate ODF-LVM function**
+
+### Implementation Details/Notes/Constraints [optional]
+
+<!---
+What are the caveats to the implementation? What are some important details that
+didn't come across above. Go in to as much detail as necessary here. This might
+be a good place to talk about core concepts and how they relate.
+--->
+
+#### API Impacts
+
+**Configuration API**
+This proposal extends the MicroShift config API with the optional lvmd configuration.  The lvmd config is not added to 
+the core MicroShift configuration, and must be written to `/etc/microshift/lvmd.yaml`
```

Comment:
You could link to our design guidelines to point out that this is consistent with how we want MicroShift to be configured.